org.eclipse.lsp4xml.extensions.maven/src/test/java/org/eclipse/lsp4xml/extensions/maven/test/PluginTest.java

```diff
@@ -75,4 +75,19 @@ public void testCompleteConfigurationParameters() throws IOException, Interrupte
 		} while (items.stream().map(CompletionItem::getTextEdit).map(TextEdit::getNewText)
 				.noneMatch("appendOutput"::equals));
```

Comment:
Now that the text edit is something like `"<parameter>$0</parameter>"`, the `TextEdit` being checked for here should be `"<appendOutput>$0</appendOutput>"` instead of `"appendOutput"` (for `testCompleteConfigurationParameters`)

Reply:
sorry, I'll change that!

Reply:
No problem :) I think it's good for a merge after that

Reply:
I just pushed a new version that checks label instead.

Reply:
Perfect